install/k8s/k8s1.8/aci_gw.yaml

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Why do we need a new copy of the yaml files if they are an exact copy of 1.6 folder?

Reply:
That's a good point. I had followed the model of having a separate folder, but if nothing is different, I agree with you.

Reply:
While it's true that the files in install/k8s/k8s1.6 and k8s1.8 are the same, the files in cluster/k8s/k8s1.6 and k8s1.8 are not the same, so it seems weird to me to in one case point to 1.6 files for 1.8, and in another case, point to 1.8 files.

Reply:
There maybe additional change for 1.8 testing. I think it is cleaner to have a separated folder for 1.8.